firmware/user/modes/mode_flight.c

```diff
@@ -135,7 +135,7 @@ static const char fl_quit[]   = "QUIT";
 
 
 
-static const int16_t IsoSphereVertices[] ICACHE_RODATA_ATTR = { 
+static const int16_t IsoSphereVertices[] RODATA_ATTR = { 
```

Comment:
@cnlohr you may want to experiment with using `int32_t` here instead of `int16_t`. Aligned reads may be a touch faster?